### PR TITLE
SCHEMA AND DATA

### PR DESCRIPTION
**Pull request by Goldinium**
- The following queries are taking too much time (**1 sec = 1000ms can be considered as too much time for database query**). 
- 
  - `SELECT COUNT(*) FROM visits where animal_id = 4;`
  - `SELECT * FROM visits where vet_id = 2;`
  - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
  
- We used `EXPLAIN ANALYZE` on the previous queries to check what is happening.
- We decreased the execution time of the first query.
- And found a way to improve execution time of the other two queries.

Screenshots, before:

Screenshots, after:

### CODE REVIEW

File: screenshots/PR4/Screenshot from 2023-11-02 11-27-04.png
Comment (binhussen) [inline screenshots/PR4/Screenshot from 2023-11-02 11-27-04.png:1]: - Kindly, Insert owners until you get >1000ms witch is dependent by your machine to fit the requirmnet 👍

File: screenshots/PR4/Screenshot from 2023-11-02 11-33-28.png
Comment (binhussen) [inline screenshots/PR4/Screenshot from 2023-11-02 11-33-28.png:1]: - Kindly, the excusion time after index has to be below  1000 ms  to fit the requirement 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Goldinium and @chiedozie013 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Added performance Audit ✔️
- Well Stractured Query ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: screenshots/PR4/Screenshot from 2023-11-02 11-33-28.png
Comment (Goldinium) [reply screenshots/PR4/Screenshot from 2023-11-02 11-33-28.png:1]: resolved

File: screenshots/PR4/Screenshot from 2023-11-02 11-27-04.png
Comment (Goldinium) [reply screenshots/PR4/Screenshot from 2023-11-02 11-27-04.png:1]: resolved

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Goldinium, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**